packages/imagelib/imagelib.20200929/opam

```diff
@@ -0,0 +1,59 @@
+synopsis: "Library implementing parsing of image formats such as PNG, BMP, PPM"
+description:
+"""
+The imagelib library implements image formats such as PNG, BMP, and PPM in
+OCaml, relying on only one external dependency: 'decompress'.
+
+Unix-dependent functionality such as reading or writing to files in the
+filesystem are packaged separately in the 'imagelib-unix' OPAM package.
+
+Supported image formats:
+ - PNG (full implementation of RFC 2083),
+ - PPM, PGM, PBM, ... (fully supported),
+ - BMP (read-only)
+ - JPG (only image size natively, conversion to PNG otherwise),
+ - GIF (only image size natively, conversion to PNG otherwise),
+ - XCF (only image size natively, conversion to PNG otherwise),
+ - Utility functions for handling unimplemented formats are available in
+   the 'imagelib-unix' OPAM package. See that package description for more info.
+
+As imagelib only requires 'decompress', it can be used together with
+js_of_ocaml to compile projects to Javascript, or from MirageOS unikernels.
+"""
+opam-version: "2.0"
+maintainer: "rodolphe.lepigre@inria.fr"
+bug-reports: "https://github.com/rlepigre/imagelib/issues"
+homepage: "https://github.com/rlepigre/imagelib"
+dev-repo: "git+https://github.com/rlepigre/imagelib.git"
```

Comment:
These links are broken. The `ocaml-` prefix is missing it seems.

Reply:
Thanks. Well spotted. I will fix them here and separately do it on the image lib repo